nss-day-cohort-41/tabloid-fullstack-elvis_spotters
# Removed activate/deactivate and promote/demote for active users account

**Details:
This is a feature fix. When you are viewing user-profiles, if you go to your detail pages, you are not able to activate/ deactivate, promote/demote yourself.
**Fixes: n/a
**Testing instructions:
Navigate to the user profiles page and selected current users active, when you navigate you should have no option of demotion, promotion, activation, or deactivation

- [X] Code is commented in hard to understand areas.
- [X] Code is within styling standards.
- [X] Testing instructions included.
